src/winstanley/wdl.bnf

```diff
@@ -24,23 +24,26 @@ import_namespace ::= AS IDENTIFIER
 
 workflow_block ::= WORKFLOW WORKFLOW_IDENTIFIER_DECL LBRACE wf_body_element* RBRACE
 wf_body_element ::= call_block | declaration | while_loop | if_stmt | scatter_block | wf_outputs
-call_block ::= CALL fqn alias? (LBRACE call_body RBRACE)?
+
+callable_lookup ::= CALL fully_qualified_name {methods=[getReferences]}
+call_block ::= callable_lookup alias? (LBRACE call_body RBRACE)?
```

Comment:
I think you could move the `CALL` back into the `call_block` line without changing anything in the scala code, but it would mean we only highlight the name when hovering over the `fqn` to go to declaration